backend/src/tasks/wappalyzer.ts

```diff
@@ -7,12 +7,16 @@ import { wappalyzer } from './helpers/simple-wappalyzer';
 const wappalyze = async (domain: LiveDomain): Promise<void> => {
   try {
     const { data, status, headers } = await axios.get(domain.url, {
+      timeout: 2000,
```

Comment:
Should we do a 5 second or 10 second timeout perhaps, just to be safe?

Reply:
I will update to 10 seconds.